### PR TITLE
Recruitech Theme Implementation

### PR DESCRIPTION
**Pull request by DiegoDA04**
Contains all values for recruitech theme as typography and colors.

### CODE REVIEW

File: app/src/main/java/pe/edu/notcodingdevs/recruitech/ui/theme/Color.kt [hunk 1]
```diff
@@ -8,4 +8,15 @@ val Pink80 = Color(0xFFEFB8C8)
 
 val Purple40 = Color(0xFF6650a4)
```
Comment (jackby03) [inline app/src/main/java/pe/edu/notcodingdevs/recruitech/ui/theme/Color.kt:9]: Delete unused values

**Review by jackby03 — COMMENTED** [review]
Arregla tu webada uwu

File: app/src/main/java/pe/edu/notcodingdevs/recruitech/ui/theme/Color.kt (hunk 1 above)
Comment (DiegoDA04) [reply app/src/main/java/pe/edu/notcodingdevs/recruitech/ui/theme/Color.kt:9]: Ta bien oe

**Review by jackby03 — APPROVED** [review]
te valio pinga mi review , nimodo 💁

**Pull request merged**